src/store/peer/reducer.ts

```diff
@@ -25,14 +28,22 @@ const initialState: StateType = {
     failure: false,
     error: null,
     data : null
+  },
+  setUpdateGroupsVisible: false,
+  savedGroups: <ChangeResponse<Group[] | null>> {
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  savedGroups: <ChangeResponse<Group[] | null>>{
```